README_OTHERPLATFORMS.md

```diff
@@ -1,97 +1,28 @@
-# Compiling for other platforms (RISC-V, ARM32, ARM64, WIN32, WIN64)
+# Cross compiling for other platforms on Linux.
 
-## 1) Environment setup (RISC-V example)
-Install a Debian-based Linux distro (choose one)
-- SiFive Unmatched: [Ubuntu Server 20.04](https://ubuntu.com/tutorials/how-to-install-ubuntu-on-risc-v-hifive-boards#1-overview)
-- Virtual Machine: [RISC-V VM with QEMU](https://colatkinson.site/linux/riscv/2021/01/27/riscv-qemu/)
+## Non-native Linux targets.
 
-## 2) Install dependencies
-``` sh
-apt update
-apt install git build-essential bc cmake python3 python3-pip python-is-python3 libffi-dev libssl-dev cargo golang ninja-build
-```
-
-## 3) Clone sources
-``` sh
-#Toit
-git clone https://github.com/toitlang/toit.git
-cd toit/
-
-#ESP-IDF
-git submodule update --init --recursive
-```
-
-## 4) Compile Toit SDK
-``` sh
-make all
-```
-
-## 5) Run examples
-``` sh
-build/host/sdk/bin/toit.run examples/hello.toit
-build/host/sdk/bin/toit.run examples/bubble_sort.toit
-build/host/sdk/bin/toit.run examples/mandelbrot.toit
-```
-</br>
-</br>
-
-# Cross-compiling
-This section describes how to compile the Toit binaries (toit.compile and toit.run) for another
-architecture (like RISC-V).
-
-## 1) Prepare host tools
->Note: This is necessary to generate the snapshots that are part of the SDK.
+You need to install clang (version 16.0.6 works).  It will fetch the
+back ends that it needs.
```

Comment:
Is that what you mean?
```suggestion
You need to install clang (version 16.0.6 works).  Our Makefile will fetch the
necessary sysroots that it needs.
```

Reply:
You could also add the list of supported sysroots (and point to our repository).